frontend/lib/common.js

```diff
@@ -9,7 +9,10 @@ import getEnvVars from '../environment';
  * BASE("Products")...
  **/
 
-const { AIRTABLE_API_KEY, AIRTABLE_BASE_ID } = getEnvVars();
+const { AIRTABLE_API_KEY, AIRTABLE_BASE_ID, IMG_API_KEY } = getEnvVars();
```

Comment:
For documentation purposes, leaving this link for image API key setup: https://medium.com/@ifeoluwaking24/how-to-upload-an-image-in-expo-react-native-to-firebase-using-cloudinary-24aac981c87